vast-example-1.ipynb

```diff
@@ -0,0 +1,108 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "logger = logging.getLogger()\n",
+    "logger.setLevel(logging.DEBUG)\n",
+    "logging.debug(\"Logger active\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyvast"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vast = pyvast.VAST(app=\"build/dbg/bin/vast\")"
```

Comment:
For this notebook to work with other setups, how about avoiding a hardcoded path? If you make sure that `vast` is in your `$PATH`, then everything should work, right?

Reply:
I generally prefer to avoid depending on a modified environment. I can add a step for that at the beginning of `demo.md` though, to make it a little more explicit.

Reply:
I see your point, but hardcoding this here is not practical, unless only one person with one specific setup uses this demo.